tools/operator/launcher/k8s/createStatefulset.go

```diff
@@ -0,0 +1,410 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package k8s
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	v12 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/hyperledger/fabric-test/tools/operator/logger"
+	"github.com/hyperledger/fabric-test/tools/operator/networkspec"
+	"github.com/hyperledger/fabric-test/tools/operator/paths"
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	//restclient "k8s.io/client-go/rest"
+	//
+	// Or uncomment to load specific auth plugins
+	// _ "k8s.io/client-go/plugin/pkg/client/auth/azure"
+	// _ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	// _ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
+	// _ "k8s.io/client-go/plugin/pkg/client/auth/openstack"
+)
+
+type configmapData struct {
+	MSP        map[string]string
+	TLS        map[string]string
+	Config     map[string]string
+	Admincerts map[string]string
+	CA         map[string]string
+}
+
+//CreateStatefulset --
+func (k K8s) CreateStatefulset(l LaunchConfig, nsConfig networkspec.Config) error {
```

Comment:
In general, a variable name should be descriptive enough to know what it is without having to look for its type, can we rename `k` to `k8s`  and take a general look at variables throughout and make sure we can derive their context from their name